js/alert.js

```diff
@@ -0,0 +1,177 @@
+let IsGettingAlertsucceed = false;
```

Comment:
```suggestion
let isGettingAlertSucceed = false;
```
Keep your variable names in `camelCase`. Be careful with this; it is not the first time I'm telling you this.